it/src/main/java/org/corfudb/universe/node/server/vm/VmCorfuServerParams.java

```diff
@@ -0,0 +1,31 @@
+package org.corfudb.universe.node.server.vm;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import org.corfudb.universe.node.server.CorfuServerParams;
+import org.slf4j.event.Level;
+
+import java.time.Duration;
+
+import static org.corfudb.universe.node.server.CorfuServer.Mode;
+import static org.corfudb.universe.node.server.CorfuServer.Persistence;
+
+
+/**
+ * Represents the parameters for constructing a {@link VmCorfuServer}.
+ */
+@Getter
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class VmCorfuServerParams extends CorfuServerParams {
+    private final String vmName;
+
+    @Builder
+    public VmCorfuServerParams(String vmName, String streamLogDir, int port, Mode mode, Persistence persistence,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)